src/pages/team.js

```diff
@@ -47,6 +91,30 @@ const TeamPage = () => {
           <p>{teamData.description}</p>
         </div>
       </div>
+
+      {/* Main content */}
+      {/* Proper team Block please god*/}
```

Comment:
I don't know what `please god` means but I'm glad that I saw it. If there is a real issue here it would be nice to have more descriptive comments.

Reply:
omg no sorry that was just me praying that it would work, forgot to delete it haha